integration_tests/models/sql/test_star_no_quotes.sql

```diff
@@ -0,0 +1,10 @@
+with data as (
+
+    select
+        {{ dbt_utils.star(from=ref('data_star_no_quotes'), quote_identifiers=False) | upper }}
+
+    from {{ ref('data_star_no_quotes') }}
+
+)
+
+select * from data
```

Comment:
Will the test fail if this is changed to `quote_identifiers=True`? I _suspect_ not! If not, then it's not really covering the test case.

I'd be happy to be wrong, so give it a shot and let me know either way.

Presuming that it's not covering the case, I'll add an idea how to handle this.

Reply:
Indeed, changing to `quote_identifiers=True` did **not** induce a test failure, but we would have expected a sufficient test to fail in this scenario. Read on for details.

When I made these changes:
https://github.com/dbt-labs/dbt-utils/compare/37a2b69..bf60046

Everything still passed:
<img width="259" alt="image" src="https://user-images.githubusercontent.com/44704949/196433646-cf9284ec-ea6a-4c29-b32d-d5b9358b65d2.png">

So you'll want to reassess the approach to testing this.

Suggestion for next steps: try the approach outlined [here](https://github.com/dbt-labs/dbt-utils/pull/706#pullrequestreview-1145006024).

Reply:
Agreed, I had to enclose the column name in quotes for it to return a failure with `quote_identifiers=True`. my thought for this one was that it would only return true because of the case sensitive column requirements.

Checking out your suggested solution though, I like the approach and appreciate your time!

Reply:
With a slight edit to your suggested approach, the new test is passing.
And, if I flip the `quote_identifiers` boolean in each of quotes/no quotes, the tests both fail this time!